scripts/runtime-test-lock.py

```
@@ -0,0 +1,69 @@
+"""Run one runtime-package test command at a time on the local host."""
+
+import os
+import stat
+import subprocess
+import sys
+import tempfile
+import time
+
+
+def open_shared_lock():
+    lock_path = "/tmp/kent-runtime-test.lock"
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/kent-runtime-test.lock"

(S108)